### PR TITLE
Confirmation dialogs

### PR DESCRIPTION
**Pull request by orange-elephant**
Draft

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -56,7 +56,7 @@ dependencies {
     implementation 'androidx.activity:activity-compose:1.7.2'
     implementation "androidx.compose.ui:ui:$compose_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
-    implementation 'androidx.compose.material3:material3:1.2.0-alpha02'
+    implementation 'androidx.compose.material3:material3:1.1.1'
```
Comment (Toadsta) [inline app/build.gradle:59]: we downgrading this package?

**Review by Toadsta — COMMENTED** [review]
if you did intent do downgrade then go for it

**Review by Toadsta — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (orange-elephant) [reply app/build.gradle:59]: yes to stable lol

Comment (orange-elephant) [reply app/build.gradle:59]: i also have no clue what navigation runtime is for? in the dependencies

**Pull request merged**